package.json

```diff
@@ -14,16 +14,17 @@
   "dependencies": {
     "@capacitor/android": "^2.4.7",
     "@capacitor/core": "^2.4.7",
-    "@ionic/core": "6.0.1",
-    "@ionic/vue": "6.0.1",
-    "@ionic/vue-router": "6.0.1",
+    "@ionic/core": "6.0.3",
```

Comment:
Why this is changed?  
I think @ymaheshwari1 fixed issue with ion-input with this change
https://github.com/hotwax/receiving/pull/50

Reply:
Yes Sir, the camera was not closing because of this so we updated it to version 6.0.3 and it also resolved the issue related to ion input.